Reviewing: if left comments, leave merging to the author (#15)
populationgenomics/team-docs: getting_started.md

"Request Changes". Otherwise, once you are happy with the changes and all
comments have been addressed, you can "Approve" the PR.

After approving, you can also merge the PR, unless you left any feedback 

> during the latest round (even minor one): in this case, it's preferrable to 
> leave merging to the author in case the author wants to act on your feedback.

Reviewer: ```suggestion
that you'd like the author to act on: in this case, it's preferable to leave merging to the author.
```

Author: I think that should work regardless of whether you wanted or not for the author to act on a comment:
- When you want the author to act on your comment, you leave the PR not approved and go for a second round;
- When you are okay with your comment left unacted upon  - it might be just a minor suggestion, or some thought, or a pointer to something related - you approve the PR, but leave it to the author to decide if they want to act on your comment.

Author: Though, doesn't really matter - I think merging is okay in the second scenario too; the author can act on comments in another PR. Anyway, clarified the paragraph a bit better. Thanks for the comments :)
